src/docs/03_system_scope_and_context.adoc

```diff
@@ -23,44 +23,63 @@ Various options:
 
 === Business Context
 
-[role="arc42help"]
-****
-.Contents
-Specification of *all* communication partners (users, IT-systems, ...) with explanations of domain specific inputs and outputs or interfaces.
-Optionally you can add domain specific formats or communication protocols.
 
-.Motivation
-All stakeholders should understand which data are exchanged with the environment of the system.
 
-.Form
-All kinds of diagrams that show the system as a black box and specify the domain interfaces to communication partners.
+image::SystemScopeDiagram.png[System scope diagram] 
 
-Alternatively (or additionally) you can use a table.
-The title of the table is the name of your system, the three columns contain the name of the communication partner, the inputs, and the outputs.
-****
 
-**<Diagram or Table>**
+.Viade
+|=========================================================
+| Communication partner |Inputs |Outputs
+
+| User
+| Upload routes. Attaches photos, videos or information to them
+| Reads routes from its own pod
```

Comment:
The user can see the routes in a map.

Reply:
Thats not an input, thats what the user does with the input